cloudformation/media-atom-maker.json

```diff
@@ -55,6 +55,14 @@
         "AuditTable": {
             "Description": "Name of the audit dynamo table",
             "Type": "String"
+        },
+        "AlertWebhook": {
+          "Description": "Where CloudWatch alerts are sent",
+          "Type": "String"
+        },
+        "AlertActive": {
```

Comment:
is an alarm that can be turned off, worthy of being an alarm?

if a tree falls in the forest...